_docs/_latest/concepts/service-accounts.md

```diff
@@ -1,8 +1,64 @@
 ---
 title: Service Accounts
-order: 100
+order: 004
 ---
 
 # {{ page.title }}
 
-TODO
+By default Forseti will create and use multiple service accounts in its default deployment. In 
+doing this Forseti implements the security best practice of privilege separation and least
+privilege.
+
+Following are the service accounts Forseti creates on your behalf.
+
+ * **[Server service account](#the-server-service-account)**
+ * **[Client service account](#the-client-service-account)**
+ 
+---
+
+**The image below shows how the default service accounts created by Forseti are used.**
+
+{% responsive_image path: images/docs/concepts/service-account-architecture.png alt: "service account architecture diagram" %}
+
+## The Server Service Account 
+
+The `forseti-server-gcp` service account has more access and is used exclusively on the
+`forseti-server-vm` virtual machine instance.
+
+This service account is used by core modules of the Forseti service. For
+example, Inventory uses this service account to read and store the supported resources. It's also
+used by Scanner to audit policies.
+
+### Permissions 
+
+For Forseti to work properly the `forseti-server-gcp` service account requires the
+following permissions.
+
+{% include docs/latest/forseti-server-gcp-required-roles.md %}
+
+## The Client Service Account 
+
+The `forseti-client-gcp` service account has less access and is used exclusively on the
+`forseti-client-vm` virtual machine instance.
+
+This service account is used to communicate with the `forseti-server-vm`. The separation 
+between service accounts is key to securing the granted rights of the `forseti-server-gpc`
```

Comment:
typo: `forseti-server-gpc` -> `forseti-server-gcp`

Reply:
Same for `forseti-client-gpc` in the line below.

Reply:
done